<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>sc-eQTLGen</title>

    <link href="css/vendor/bootstrap/bootstrap.min.css" rel="stylesheet">
    <!-- <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.css"> -->
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css">
    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
    <link href="css/eqtlgen.css" rel="stylesheet">

  </head>

  <body>

    <!-- Navigation -->
    <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark fixed-top">
      <div class="container">
        <a class="navbar-brand" href="single-cell.html">sc-eQTLGen</a>
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
<!--         <a class="navbar-brand" href="http://www.eqtlgen.org">eQTLGen</a>
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button> -->
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="participating.html">Consortium members</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="contact.html">Contact</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <!-- Page Content -->
    <div class="container" style="margin-top: 75px; max-width:700px">

      <h2 class="my-4">Consortium members</h2>
      <div class="text-center">
        <img src="img/lude_franke.jpeg" class="text-center" align="left" style="width:100%; height:100%; max-width:300px;padding-right:30px;padding-bottom: 20px"/>
      </div>
      <p align="justify">
        <b>Dr. Lude Franke, dept. of Genetics, UMCG (Groningen, the Netherlands)</b>
        <br />
        <u>Study population:</u> general population of the Netherlands (20-80 years old)
        <br />
        <u>Sample:</u> PBMCs, before and after <i>Plasmodium infection</i>
        <br />
        <br />

        Our group studies the mechanisms and pathways through which pathogenic variants cause disease by correlating genetic variation with molecular data (such as gene expression, methylation, and histone modifications) in thousands of individuals. Single-cell RNA-seq data now gives us the opportunity to identify personalized effects on gene regulatory networks and pinpoint disease-relevant cell types.
      </p>
      <div class="text-center" style="padding-top:25px">
        <img src="img/youssef_idaghdour.png" class="text-center" align="right" style="width:95%; height:95%; max-width:300px;padding-left:30px;padding-bottom: 20px"/>
      </div>
      <p align="justify">
        <b>Dr. Youssef Idaghdour, dept. of Biology, NYUAD (Abu Dhabi, United Arab Emirates)</b>
        <br />
        <u>Study population:</u> West African children from Burkina Faso (2-10 years old)
        <br />
        <u>Sample:</u> unstimulated and pathogen-stimulated PBMCs
        <br />
        <br />
        The overarching goal of our group is to understand how genomes and environment interact moving from the architecture of the genome itself to protein function and transitioning through the complex modes of epigenetic and gene expression regulation. We are particularly interested in studying these effects in vivo and focus on metabolic traits and immune response to malaria infection.
      </p>
      <div class="text-center" style="padding-top:25px">
        <img src="img/pim_van_der_harst.png" class="text-center" align="left" style="width:100%; height:100%; max-width:300px;padding-right:30px;padding-bottom: 20px"/>
      </div>
      <p align="justify">
        <b>Prof. Pim van der Harst, dept. Of Cardiology, <br />UMCG (Groningen, the Netherlands)</b>
        <br />
        <u>Study population:</u> patients presenting with first ST-elevation myocardial infarction (STEMI) (40 â€“ 80 years old)
        <br />
        <u>Sample:</u> PBMCs from STEMI patients 
        <br />
        <br />
        Our group aims to study the role of PBMCs in STEMI patients. Ultimately, this could lead to new disease concepts, novel therapeutic prospects and more patient-tailored therapy.
      </p>
      <div class="text-center" style="padding-top:100px">
        <img src="img/joseph_powell.png" class="text-center" align="right" style="width:95%; height:95%; max-width:300px;padding-left:30px;padding-bottom: 20px"/>
      </div>
      <p align="justify">
        <b>Associate Professor Joseph Powell, Head | Garvan-Weizmann Centre for Cellular Genomics, Garvan Institute of Medical Research (Sydney, Australia)</b>
        <br />
        <u>Study population:</u> OneK1k: healthy aged population of Australians of N European descent (60-80 years old)
        <br />
        <u>Sample:</u> unstimulated PBMCs, genotypes, e-medical record
        <br />
        <br />
        <br />
        Our group focuses on demonstrating the genomic mechanisms by which loci contribute to complex human diseases, and working towards early stage diagnosis methods and targeted therapeutics. To do so, we apply existing computational approaches, and develop our own statistical genetics methods for analysis of large-scale next generation sequencing data. Following in silico experiments, we perform functional validation of statistical observations using molecular techniques such as high-throughput genome editing and cell phenotyping.
      </p>
      <div class="text-center" style="padding-top:25px">
        <img src="img/chun_jimmie_ye.jpg" class="text-center" align="left" style="width:100%; height:100%; max-width:300px;padding-right:30px;padding-bottom: 20px"/>
      </div>
      <p align="justify">
        <b>Dr. Chun Jimmie Ye, dept. Of Epidemiology and Biostatistics, UCSF (San Francisco, USA)</b>
        <br />
        <u>Study population:</u> Female of mostly European descent
        <br />
        <u>Sample:</u> PBMCs from lupus patients and healthy controls
        <br />
        <br />
        We are interested in how genetics and environment interact to affect molecular phenotypes. The general strategy is to couple high-throughput sequencing with population genetics to measure and model cellular response to environmental challenges across large patient cohorts. The lab develops experimental approaches that enable the collection of functional genomic data en masse and computational methods that translate the data into biological insights. The initial focus is to study human immune cells in healthy and diseased patients to understand host pathogen interactions and its role in autoimmunity.
      </p>
      <div class="text-center" style="padding-top:100px">
        <img src="img/martijn_nawijn.png" class="text-center" align="right" style="width:95%; height:95%; max-width:300px;padding-left:30px;padding-bottom: 20px"/>
      </div>
      <p align="justify">
        <b>Associate Professor Martijn Nawijn, Groningen Research Institute for Asthma and COPD (GRIAC), UMCG (Groningen, the Netherlands)</b>
        <br />
        <u>Study population:</u> Individuals with chronic respiratory disease (asthma, COPD), and matched healthy controls
        <br />
        <u>Sample:</u> FACS-purified CD4 T cells from peripheral blood and matched airway wall biopsies
        <br />
        <br />
        <br />
        Our group focuses on understanding genetic mechanisms governing the immunological and (structural) cell biological alterations in airway wall that contribute to susceptibility for chronic respiratory disease. We aim to elucidate individual pathways leading to the inception, chronicity and exacerbations of asthma and COPD, paving the way for personalized diagnosis and treatment of disease. 
      </p>

    </div>
    <!-- /.container -->

    <!-- Footer -->
    <footer class="pt-5">
      <hr>
      <div class="container text-center">
        <img src="img/rug.jpg" height="100px">
        <p class="m-0 text-center text-white">Copyright &copy; eQTLGen 2018</p>
      </div>
      <!-- /.container -->
    </footer>

    <script src="vendor/jquery/jquery.min.js"></script>
    <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>
    <script>
      //https://eqtl.molgenis.org/api/v2/eqtlgen_eqts
      $(document).ready( function () {

        var columns = ["pvalue", "SNP", "SNP.Chr", "SNP.Pos", "gene", "gene.Symbol", "gene.Chr", "gene.Pos", "assessed_allele", "other_allele", "nr_cohorts", "nr_samples", "FDR"]

        $('#eqtl-table').DataTable( {
          serverSide: true,
          processing: true,
          language: { processing: "<i class='fa fa-spinner fa-spin fa-3x fa-fw'></i>" },
          dom: "<'row'<'col-sm-12 col-md-6'f><'col-sm-12 col-md-6'l>>" +
                "<'row'<'col-sm-12'tr>>" +
                "<'row'<'col-sm-12 col-md-5'i><'col-sm-12 col-md-7'p>>",
          scrollX: true,
          columnDefs: [ {
            "targets": [2,3,5,6,7],
            "orderable": false
          } ],
          ajax: {
            url: 'https://eqtl.molgenis.org/api/v2/eqtlgen_trans_eqtl',
            "data": function (d) {

              console.log(d);

              var request = {};
              request.num = d.length
              request.start = d.start
              request.sort = columns[d.order[0].column] + ':' + d.order[0].dir
              request.attrs = "pvalue,gene(*),SNP(*),zscore,FDR,nr_samples,nr_cohorts,assessed_allele,other_allele"
              if (d.search.value) {
                var searchUrlSafe = encodeURI(d.search.value)
                request.q = 'gene.ID=like=' + searchUrlSafe + ',SNP.ID=like=' + searchUrlSafe + ',gene.Symbol=like=' + searchUrlSafe
              }
              return(request)
            },
            dataFilter: function(data) {
              var json = jQuery.parseJSON( data );
              json.recordsTotal = json.total;
              json.recordsFiltered = json.total;

              json.data = [];
              for (var i=0, ien=json.items.length ; i<ien ; i++) {
                row = [];

                row.push(json.items[i].pvalue);
                row.push(json.items[i].SNP.ID);
                row.push(json.items[i].SNP.Chr);
                row.push(json.items[i].SNP.Pos);
                row.push(json.items[i].gene.ID);
                row.push(json.items[i].gene.Symbol);
                row.push(json.items[i].gene.Chr);
                row.push(json.items[i].gene.Pos);
                row.push(json.items[i].assessed_allele);
                row.push(json.items[i].other_allele);
                row.push(json.items[i].nr_cohorts);
                row.push(json.items[i].nr_samples);
                row.push(json.items[i].FDR);

                json.data.push(row);
              }

              return JSON.stringify( json ); // return JSON string
            }
          }
        });
      });
    </script>

  </body>

</html>
