<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>eQTLGen - Publications</title>

    <!-- Bootstrap core CSS -->
    <link href="css/vendor/bootstrap/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/eqtlgen.css" rel="stylesheet">

  </head>

  <body>

    <!-- Navigation -->
    <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark fixed-top">
      <div class="container">
        <a class="navbar-brand" href="index.html">eQTLGen</a>
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="cis-eqtls.html"><em>Cis</em>-eQTLs</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="trans-eqtls.html"><em>Trans</em>-eQTLs</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="eqts.html">eQTS</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="replications.html">Replications</a>
            </li>
            <li class="nav-item">
              <a class="nav-link active" href="publications.html">Publications</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <!-- Page Content -->
    <div class="container" style="margin-top: 75px">

      <h2 class="my-4">Publications</h2>

      An overview of papers, tools and databases using eQTLGen Consortium data.<br/>
      Last update: 26 March 2019.<br/><br/>
      <h5>Papers and preprints:</h5>

      <ul>
  			<li>Lepik, K., et al. (2017). C-reactive protein upregulates the whole blood expression of CD59 - an integrative analysis. PLOS Computational Biology, 13(9), e1005766. <a href="http://doi.org/10.1371/journal.pcbi.1005766">http://doi.org/10.1371/journal.pcbi.1005766</a></li>
  			<li>Xue, A., et al. (2018). Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. Nature Communications, 9(1), 2941. <a href="http://doi.org/10.1038/s41467-018-04951-w">http://doi.org/10.1038/s41467-018-04951-w</a></li>
  			<li>Qi, T., et al. (2018). Identifying gene targets for brain-related traits using transcriptomic and methylomic data from blood. Nature Communications, 9(1), 2282. <a href="http://doi.org/10.1038/s41467-018-04558-1">http://doi.org/10.1038/s41467-018-04558-1</a></li>
  			<li>Wray, N. R., et al. (2018). Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. Nature Genetics, 50(5), 668–681. <a href="http://doi.org/10.1038/s41588-018-0090-3">http://doi.org/10.1038/s41588-018-0090-3</a></li>
  			<li>Timmers, P. R., et al. (2018). Genomic underpinnings of lifespan allow prediction and reveal basis in modern risks. BioRxiv, 363036. <a href="http://doi.org/10.1101/363036">http://doi.org/10.1101/363036</a></li>
  			<li>Karlsson Linnér, R., et al. (2019). Genome-wide association analyses of risk tolerance and risky behaviors in over 1 million individuals identify hundreds of loci and shared genetic influences. Nature Genetics, 51(2), 245–257. <a href="http://doi.org/10.1038/s41588-018-0309-3">http://doi.org/10.1038/s41588-018-0309-3</a></li>
  			<li>Timmers, P. R., et al. (2019). Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. ELife, 8. <a href="http://doi.org/10.7554/eLife.39856">http://doi.org/10.7554/eLife.39856</a></li>
  			<li>Thompson, D., et al. (2019). Genetic predisposition to mosaic Y chromosome loss in blood is associated with genomic instability in other tissues and susceptibility to non-haematological cancers. BioRxiv, 514026. <a href="http://doi.org/10.1101/514026">http://doi.org/10.1101/514026</a></li>
  			<li>Porcu, E., et al. (2019). Mendelian Randomization integrating GWAS and eQTL data reveals genetic determinants of complex and clinical traits. BioRxiv, 377267. <a href="http://doi.org/10.1101/377267">http://doi.org/10.1101/377267</a></li>
  			<li>Nath, A. P., et al. (2019). Multivariate genome-wide association analysis of a cytokine network reveals variants with widespread immune, haematological and cardiometabolic pleiotropy. BioRxiv, 544445. <a href="http://doi.org/10.1101/544445">http://doi.org/10.1101/544445</a></li>
  			<li>Madan, N., et al. (2019). Identification of functional variants for platelet CD36 expression by Massively Parallel Reporter Assay. BioRxiv, 550871. <a href="http://doi.org/10.1101/550871">http://doi.org/10.1101/550871</a></li>
  			<li>Richardson, T. G., et al. (2019). A transcriptome-wide Mendelian randomization study to uncover tissue-dependent regulatory mechanisms across the human phenome. BioRxiv, 563379. <a href="http://doi.org/10.1101/563379">http://doi.org/10.1101/563379</a></li>
  			<li>Hillary, R. F., et al. (2019). Genetic and epigenetic architectures of neurological protein biomarkers in the Lothian Birth Cohort 1936. BioRxiv, 558940. <a href="http://doi.org/10.1101/558940">http://doi.org/10.1101/558940</a></li>
  			<li>Nalls, M. A., et al. (2019). Expanding Parkinson’s disease genetics: novel risk loci, genomic context, causal insights and heritable risk. BioRxiv, 388165. <a href="http://doi.org/10.1101/388165">http://doi.org/10.1101/388165</a></li>
  			<li>Zwir, I., et al. (2019). Uncovering the complex genetics of human personality: response from authors on the PGMRA Model. Molecular Psychiatry, 1. <a href="http://doi.org/10.1038/s41380-019-0399-z">http://doi.org/10.1038/s41380-019-0399-z</a></li>
  			<li>Cai, X.-Y., et al. (2019). GWAS Follow-up Study Discovers a Novel Genetic Signal on 10q21.2 for Atopic Dermatitis in Chinese Han Population. Frontiers in Genetics, 10, 174. <a href="http://doi.org/10.3389/fgene.2019.00174">http://doi.org/10.3389/fgene.2019.00174</a></li>
  			<li>Lappalainen, T., et al. (2019). Genomic Analysis in the Age of Human Genome Sequencing. Cell, 177(1), 70–84. <a href="http://doi.org/10.1016/j.cell.2019.02.032">http://doi.org/10.1016/j.cell.2019.02.032</a></li>
      </ul>

      <h5>Computational methods</h5>
      Online computational methods that use eQTLGen data and came available after submitting our initial manuscript:<br/>
      <ul>
        <li><a target="_blank" href="http://fuma.ctglab.nl/">http://fuma.ctglab.nl/</a></li>
        <li><a target="_blank" href="http://locuscompare.com/">http://locuscompare.com/</a></li>
        <li><a target="_blank" href="https://kalays01.u.hpc.mssm.edu/hipc/">https://kalays01.u.hpc.mssm.edu/hipc/</a></li>
        <li><a target="_blank" href="http://mrcieu.mrsoftware.org/Tissue_MR_atlas/">http://mrcieu.mrsoftware.org/Tissue_MR_atlas/</a></li>
      </ul>

    </div>
    <!-- /.container -->

    <!-- Footer -->
    <footer class="pt-5">
      <hr>
      <div class="container text-center">
        <img src="img/rug.jpg" height="100px">
        <p class="m-0 text-center text-white">Copyright &copy; eQTLGen 2018</p>
      </div>
      <!-- /.container -->
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="js/vendor/jquery/jquery.min.js"></script>
    <script src="js/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  </body>

</html>
